higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -222,12 +222,23 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
   {{ campuses|json_script:"campus-data" }}
 {% endcache %}
 
-
-
 {% compress js inline %}
   <script type="text/javascript">
   "use strict";
 
+  let universities = JSON.parse(document.getElementById('universities-data').textContent),
+  campuses = JSON.parse(document.getElementById('campus-data').textContent);
+
+  // with the assumption other will be the only value without a province
+  // otherwise we will have to add a name to the universities json blob and do a check against it
+  let other_university = null;
+  for(const [university, prov] of Object.entries(universities)){
```

Comment:
I think this syntax is only supported on modern browsers, and we don't have any babel translation on this, so I think we should rather use the older for loop syntax here